hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/sink/utils/CompactFunctionWrapper.java

```diff
@@ -102,11 +108,10 @@ public void openFunction() throws Exception {
 
   public void compact(long checkpointID) throws Exception {
     // collect the CompactEvents.
-    CollectorOutput<CompactionPlanEvent> output = new CollectorOutput<>();
-    compactionPlanOperator.setOutput(output);
+    compactionPlanOperator.setOutput(planEventOutput);
```

Comment:
why this change?

Reply:
to get RuntimeContext in `CompactionPlanOperator` in test. Current implementation will return null when we call `getRuntimeContext`